tests/__init__.py

```diff
@@ -0,0 +1 @@
+"""Tests."""
```

Comment:
We could also ignore the ruff `D` rules in `tests`, afaik.

Reply:
Sure.

Reply:
Done.